echopype/clean/api.py

```diff
@@ -65,7 +75,7 @@ def mask_transient_noise(
     transient noise masking and translated into xarray code:
     https://github.com/open-ocean-sounding/echopy/blob/master/echopy/processing/mask_transient.py # noqa
     """
-    if "depth" not in ds_Sv.data_vars:
+    if "depth" not in ds_Sv.data_vars and not use_index_binning:
```

Comment:
I think we need to allow both `echo_range` and `depth`, since direct output from `compute_Sv` has only `echo_range` unless people explicitly want to add a depth variable to it (and we know that is a can of worm). The units are the same between `echo_range` and `depth` so that should be relatively straightforward.

Reply:
I agree I can add that